types/react/test/cssProperties.tsx

```diff
@@ -46,3 +46,6 @@ const strokeOpacityStyleTest = <svg style={strokeOpacityStyle} />;
 
 const strokeWidthStyle: React.CSSProperties = { strokeWidth: "10px" };
 const strokeWidthStyleTest = <svg style={strokeWidthStyle} />;
+
+const propertyWidth: React.CSS.Property.Width = "10px";
```

Comment:
Why not use `React.CSSProperties['width']`?

Reply:
This is because this PR tested the method by directly exporting CSS from "csstype" among many other methods.

Reply:
But why can't you use existing APIs here?

Reply:
Sorry. I missed the notification so I checked it now.
Are you sure you're talking about this?:`CSSProperties["index-sig"]`
- The namespace `Property` is exported from `csstype`, not imported from `@types/React`.
- So this cannot be used by developers during react project development. (if they don't `npm install -D csstype`).
- CSSProperties extends CSS.Properties. So basically, when a developer writes a style, the IDE sees the type of each property as Property.xxx.
Therefore, the issue is whether to use the Property namespace, or whether to export the appropriate type for each style property from React.

Of course, I can use the existing API, but the test must be based on what I wrote (export CSS), so I wrote it like this.
Should I write it in a separate file? Or how should I solve the problem?
As a result of looking at the comments, I understand that it is intentional not to access each style property through the index signature.

Reply:
> Is there any reason why the tests for my PR should use CSSProperties["foo"]?

The question is why we should introduce this new API in the first place. If the test can use `React.CSSProperties['width']` instead, it's not actually testing what's accomplished with this change.

Reply:
I apologize for the late reply. Unfortunately, I wasn't receiving notifications about this conversation.
After thinking about it, I thought it would be better to keep your opinion and the status quo.
I received a notification that this issue will be closed soon, so I think it would be better to leave it closed as is.
Next time, I will write meaningful issues and PRs.